bff-service/src/middlewares/cache.middleware.ts

```diff
@@ -0,0 +1,21 @@
+import { Injectable, NestMiddleware } from '@nestjs/common';
+import { ConfigService } from '@nestjs/config';
+import { Request, Response } from 'express';
+import { CacheRequestService } from 'src/services/cache-request.service';
+
+@Injectable()
+export class CacheMiddleware implements NestMiddleware {
+  constructor(private readonly cacheService: CacheRequestService) {}
+
+  async use(req: Request, res: Response, next: Function) {
+    if (req.method === 'GET' && req.originalUrl === '/product/products') {
```

Comment:
Hi @SkorykV , i would like to suggest extract some information as a const - it is used several times + for example new api can be added to cache, and it will be better to do if it extracted somewhere

Also, if you want you can check caching for 
http://skorykv-bff-api-development.eu-west-1.elasticbeanstalk.com/product/products/ - will not work
http://skorykv-bff-api-development.eu-west-1.elasticbeanstalk.com/product/products

Reply:
thanks. I just initially implemented caching for any GET request, and then noticed, that we need to cache only request for products. So this change was made in rush) Thanks for pointing - I`ll fix this problem with trailing slash :)